manifester/manifester.py

```diff
@@ -64,6 +86,16 @@ def create_subscription_allocation(self):
             cmd_args=[f"{self.allocations_url}"],
             cmd_kwargs=allocation_data,
         ).json()
+        logger.debug(
+            f"Received response {self.allocation} when attempting to create allocation."
+        )
+        if ("error" in self.allocation.keys() and 
+            "invalid version" in self.allocation['error'].values()):
+            raise ValueError(
+                                f"{self.sat_version} is not a valid version number."
+                                "Versions must be in the form of \"sat-X.Y\". Only released"
+                                "versions of Satellite are accepted."
```

Comment:
Alternatively, you could state that it isn't in the list of accepted versions and print those out. That could be a bit much, so your call.

Reply:
I don't think that would be too much. Better to be explicit, I'd say, and I can easily reference the `valid_sat_versions` property here.